src/test/java/bio/terra/common/SynapseUtils.java

```diff
@@ -46,20 +57,246 @@ public class SynapseUtils {
           .setEmail("dataset@connected.com")
           .setToken("token")
           .build();
-  private static final String INGEST_REQUEST_SCOPED_CREDENTIAL_PREFIX = "irsas_";
-  private static final String DESTINATION_SCOPED_CREDENTIAL_PREFIX = "dsas_";
-  private static final String SNAPSHOT_SCOPED_CREDENTIAL_PREFIX = "ssas_";
-  private static final String INGEST_REQUEST_DATA_SOURCE_PREFIX = "irds_";
-  private static final String DESTINATION_DATA_SOURCE_PREFIX = "dds_";
-  private static final String SNAPSHOT_DATA_SOURCE_PREFIX = "sds_";
-  private static final String TABLE_NAME_PREFIX = "ingest_";
   private static final String SCRATCH_TABLE_NAME_PREFIX = "scratch_";
+  private List<String> tableNames;
+  private Map<String, AzureStorageAccountResource> parquetFileNames;
+  private List<String> dataSources;
+  private List<String> scopedCredentials;
+  private List<String> storageAccountIds;
+  private AzureResourceManager client;
+  private BillingProfileModel billingProfile;
+  private AzureApplicationDeploymentResource applicationResource;
+  private static final String MANAGED_RESOURCE_GROUP_NAME = "mrg-tdr-dev-preview-20210802154510";
```

Comment:
Should this come from config?

Reply:
I guess it's already hard coded.  But at some point maybe we should consider that

Reply:
Good point - I've moved this to the connected test config.